src/main/java/edu/ucsb/cs156/organic/controllers/UsersController.java

```diff
@@ -34,4 +48,20 @@ public ResponseEntity<String> users()
         String body = mapper.writeValueAsString(users);
         return ResponseEntity.ok().body(body);
     }
+
+    @Operation(summary= "Toggle a user's instructor status")
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @PostMapping("/toggleInstructor")
+    public User postUsersToggleInstructor(
+            @Parameter(name="githubId") @RequestParam Integer githubId)
+            {
+
+        User user = userRepository.findByGithubId(githubId)
+                .orElseThrow(() -> new EntityNotFoundException(User.class, githubId));
+
+        user.setInstructor(!user.isInstructor());
+
+        User savedUser = userRepository.save(user);
+        return savedUser;
```

Comment:
The example [here](https://github.com/ucsb-cs156/proj-gauchoride/blob/0774df1d094692ae429cd4e925901ffde3a3ade8/src/main/java/edu/ucsb/cs156/gauchoride/controllers/UsersController.java#L72C1-L83C1) returns something different. Im not sure if this is required or not. @shuyiwan what are your thoughts?